packages/cli/src/commands/deploy/index.ts

```diff
@@ -453,6 +453,15 @@ export default async (client: Client) => {
   let deployStamp = stamp();
   let deployment = null;
 
+  const localConfigurationOverrides = {
```

Comment:
I added a type here and it demonstrated a bug.
```suggestion
  const localConfigurationOverrides: LocalConfiguration = {
```

Basically the `commandForIgnoringBuildStep` is used here and then the object is passed to `editProjectSettings()` which is expecting `ignoreCommand` instead

Reply:
Great catch; I think I'm actually able to simplify things significantly now too

Reply:
Okay latest commit simplified things quite a bit. I will ping when integration tests all pass